fuse/data/tokenizers/modular_tokenizer/inject_utils.py

```diff
@@ -147,10 +144,12 @@ def prepare_info_for_model_step(
 
 
         """
-        scalars_indices = []
-        scalars_values = []
-        scalars_masked_indices = []
-        prev_index_end = -1
+        ## both `all_scalars_values` and `all_scalars_valid_mask` will contain torch tensors, which will be concatanated in the end of this function
+        all_scalars_values = (
```

Comment:
all_scalar_values = [] ?

Reply:
I think that static code auto "fixed" this, due to the comment length
(I didn't write it like this)

Reply:
I moved the comments to one line above the code, so it won't do that.